src/ui/Quickview/Quickview.ts

```diff
@@ -453,10 +464,64 @@ export class Quickview extends Component {
         body: this.element.ownerDocument.body,
         sizeMod: 'big'
       });
+      this.makeModalboxAccessible(this.modalbox.modalBox);
       return computedModalBoxContent;
     });
   }
 
+  private getFocusableElements(container: HTMLElement) {
+    return _.sortBy(container.querySelectorAll<HTMLElement>('[tabindex]'), element => element.tabIndex);
+  }
+
+  private createFocusTrap(element: HTMLElement) {
+    document.addEventListener('focusin', e => {
+      if (!this.modalbox) {
+        return;
+      }
+      if (!element.contains(e.target as HTMLElement)) {
+        const focusableElements = this.getFocusableElements(element);
```

Comment:
What if there's no focusable elements returned ?

Reply:
Since the focus trap is currently only implemented for the `Quickview`, it always has at-least one focusable element (the close button). However, I agree that it would be a good idea to add a check here. I'll add one.